frontend/src/features/properties/components/forms/ParcelDetailForm.scss

```diff
@@ -107,6 +107,7 @@
     }
     .form-label {
       padding: 0 1rem 0 0;
+      min-width: 4.5rem;
```

Comment:
This will cause the label to be put on the row above the input if there is not enough space.